core/unix/include/sigcontext.h

```diff
@@ -43,7 +31,7 @@
  * extended state information in the memory layout pointed by the fpstate
  * pointer in sigcontext.
  */
-struct _fpx_sw_bytes {
+struct kernel_fpx_sw_bytes {
```

Comment:
Hmm, if we're going to change the names, we may want to match our style and have _t and a typedef.

Ditto below.